SocketRocket/Internal/SRConstants.m

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright (c) 2016-present, Facebook, Inc.
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
+//
+
+#import "SRConstants.h"
+
+size_t SRDefaultBufferSize(void) {
+    static size_t size;
+    static dispatch_once_t onceToken;
```

Comment:
If we really cared, this could be done without a dispatch_once. We have a guarantee (at least on all arch that we support) that all loads/stores of WORD size are guaranteed to be atomic, so because the initializer is so lightweight (`getpagesize()` should just be a single instruction on our targets), as an uber-performance improvement, you could simply do the following:

```
static size_t size = 0;
if (size == 0) {
    size = getpagesize();
}
```

This works because of two things -
`size_t` is never greater than the word size of the memory bus.
`getpagesize()` will always return the same value, and super fast.

Now, honestly? I don't think this matters. Just something super small I noticed.